perllib/FixMyStreet/Cobrand/Buckinghamshire.pm

```diff
@@ -703,14 +709,20 @@ sub area_ids_for_problems {
     return ($self->council_area_id, @{$self->_parish_ids});
 }
 
+sub parish_bodies {
+    my ($self) = @_;
+
+    return FixMyStreet::DB->resultset('Body')->search(
+        { 'body_areas.area_id' => $self->_parish_ids, },
+        { join => 'body_areas', order_by => 'name' }
+    );
+}
+
 # Show parish problems on the cobrand.
 sub problems_restriction_bodies {
     my ($self) = @_;
 
-    my @parishes = FixMyStreet::DB->resultset('Body')->search(
-        { 'body_areas.area_id' => $self->_parish_ids, },
-        { join => 'body_areas' }
-    )->all;
+    my @parishes = $self->parish_bodies->all;
```

Comment:
Just noticing from what you've put above, but should this be `->active` as well?

Reply:
Ah yes, it probably should! I've updated `sub parish_bodies` to always call `->active`.